ccmlib/node.py

```diff
@@ -1418,10 +1490,48 @@ def drain(self, block_on_log=False):
         if block_on_log:
             self.watch_log_for("DRAINED", from_mark=mark)
 
-    def repair(self, options=[], **kwargs):
-        args = ["repair"] + options
-        cmd = ' '.join(args)
-        return self.nodetool(cmd, **kwargs)
+    @property
+    def scylla_yaml(self) -> Dict[str, Any]:
+        return YAML().load(Path(self.get_conf_dir()) / common.SCYLLA_CONF)
+
+    @property
+    def api_port(self) -> int:
+        return self.scylla_yaml.get('api_port', 10000)
+
+    def run_repair(self, options, timeout=None):
+        return self.nodetool("repair " + options, timeout=timeout)
+
+    def run_cluster_repair(self, options, timeout=None):
+        return self.nodetool("cluster repair " + options, timeout=timeout)
+
+    def repair(self, keyspace: Optional[str] = None, tables: Optional[List[str]] = None, hosts: Optional[List[str]] = None, dcs: Optional[List[str]] = None,
+               end_token: Optional[int] = None, local: bool = False, partitioner_range: bool = False, start_token: Optional[int] = None,
+               tablet_tokens: Optional[List[str]] = None, timeout: Optional[int] = None):
+        outs_and_errs = []
```

Comment:
I think, all of this implementation should move to scylla_node.py

it doesn't have any sense being in this file/class.